python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -118,8 +118,20 @@ Parameters to use when loading a file to a Pandas dataframe
        :start-after: [START load_file_example_6]
        :end-before: [END load_file_example_6]
 
-#. **load_options** - :ref:`load_options`
+#. **load_options** - Use :ref:`load_options` to configure how the SDK loads data from your file to the dataframe.
+
+   .. note::
+
+        If the Astro SDK fails to automatically load data from your file to a dataframe, configure `Link text <link URL>` in :ref:`load_options` manually specify the schema for the dataframe.
```

Comment:
@jwitz was this `Link text <link URL>` intentional?

It seems our checks failed, with error:
/home/runner/work/astro-sdk/astro-sdk/python-sdk/docs/astro/sql/operators/load_file.rst:130:Unexpected indentation.

Would you like help on checking this?